### PR TITLE
some fix in controller profile, login/register view update, change em…

### PR DESCRIPTION
**Pull request by jonataasgoomes**
…ail and password update, cancel account update

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,7 +14,6 @@
 //= require rails-ujs
 //= require bootstrap.growl
 //= require activestorage
-//= require turbolinks
```
Comment (jhosoume) [inline app/assets/javascripts/application.js]: Can we keep turbolinks? It improves speed!

File: app/models/user.rb [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+  
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  has_one :profile_user
-  accepts_nested_attributes_for :profile_user
+
+  def nested_attributes
+  	if nested_attributes_blank?
+  		errors.add(:base, "É necessário preencher os dados pessoais")
```
Comment (jhosoume) [inline app/models/user.rb]: Incorrect indentation

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (jhosoume) [inline app/assets/javascripts/application.js]: Can we keep turbolinks? It improves speed!

Comment (jonataasgoomes) [reply app/assets/javascripts/application.js]: sure

File: app/models/user.rb (hunk 2 above)
Comment (jonataasgoomes) [reply app/models/user.rb]: thank you

**Pull request merged**